data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+			choice
+				`	"Of course. I'll show them the way."`
+					goto accept
+				`	"What do you expect to find there? Spare parts?"`
+					goto parts
+				`	"Well, I think they're probably good to get there on their own. Sorry, but I'm not interested."`
+					decline
+			label parts
+			`	"Too advanced, the composites that make up this and all other ringworlds are. Replicate them, we cannot. Not yet. If how this one was destroyed, we learn, maybe how to construct parts of our own, we could."`
+			choice
+				`	"Alright, I'll show them the way."`
+				`	"I'm no megastructure engineer. It won't help to have me go there. They can go on their own, I'm not interested."`
+					decline
+			label accept
+			`	"Excellent. Equipped with a jump drive, the ship is, and to reach this "Ruin" planet, enough fuel, it has. A few last scanners, the scientists have requested, so after quick stop at <destination>, to the ringworld, you may bring them."`
+			`	He introduces you to the Punisher's crew and the dozens of scientists and engineers, who will also be accompanying you, then he wishes you all good luck. Once the consul leaves, some among the Punisher's crew start fidgeting. Their Captain, an Arach, has them ready the ship to follow yours, and they prepare to depart.`
+				accept
+	npc accompany save
+		personality escort heroic opportunistic
+		government "Heliarch"
+		ship "Heliarch Punisher (Fuel)" "Eldest Inspector"
+
+mission "Heliarch Expedition 4"
+	name "Extraterritorial Venture"
+	description "Escort a Heliarch ship through human space and into the wormhole in Deneb, so that they can further investigate the broken ringworld."
+	landing
+	source "Ablub's Invention"
+	destination "Ruin"
+	to offer
+		has "Heliarch Expedition 3: done"
+	on offer
+		conversation
+			`The Punisher's captain has some equipment loaded onto the ship, and the mechanics do one last maintenance check on the ship before it leaves home territory. The captain tells you they will follow your lead when you launch. Every other minute, he looks nervously at the sky.`
+				accept
+	npc accompany save
+		personality escort heroic opportunistic
+		government "Heliarch"
+		ship "Heliarch Punisher (Fuel)" "Eldest Inspector"
+	on visit
+		dialog `You've reached <planet> but left the Heliarch ship behind. Best depart and wait for them to arrive.`
+	on complete
+		conversation
+			`The Punisher's captain had you follow his ship all over the planet after they refueled in the planet's odd facilities. Apparently they want an isolated place to set up their camp here, somewhere they wouldn't be easily spotted.`
+			`	They settle for a cavern on the way up a mountain, overlooking a foggy valley. A river is faintly visible through the dense fog. As they unload the cargo and start setting up, the captain comes to you and says that they'll meet you in the spaceport in a few hours to discuss "the acquisition of supplies."`
```

Comment:
Actually: question, are they aware that for this entire story arc they're a) travelling through human space and quite visible, and b) visiting somewhere that humans could easily use one of their acquired jump drives to follow them and have a look at this advanced alien warship and crew that's just come by to nosey in their space?

Reply:
Maybe add a quest where you fetch them a freighter they can modify and use as a ship in human space to this quest line.

Alternativly in the remnant cognizance storyline, it is established that replicas of every ship to have ever graced popular media are in use. So that excuse can maybe be reused.

Reply:
They don't really mind humans knowing about them, and though they really are worried about the possibility being attacked/followed, it's certainly not humans they're worried about.
I mean, I wouldn't worry about humans if I were in a Punisher.